jumanji/environments/routing/cleaner/env_test.py

```diff
@@ -19,15 +19,15 @@
 
 from jumanji.environments.routing.cleaner.constants import CLEAN, DIRTY, WALL
 from jumanji.environments.routing.cleaner.env import Cleaner
-from jumanji.environments.routing.cleaner.generator import Generator, Maze
+from jumanji.environments.routing.cleaner.generator import Generator
 from jumanji.environments.routing.cleaner.types import Observation, State
 from jumanji.testing.env_not_smoke import check_env_does_not_smoke
 from jumanji.testing.pytrees import assert_is_jax_array_tree
 from jumanji.types import StepType, TimeStep
 
 SAMPLE_GRID = jnp.array(
     [
-        [DIRTY, DIRTY, WALL, DIRTY, DIRTY],
+        [CLEAN, DIRTY, WALL, DIRTY, DIRTY],
```

Comment:
This is because the top-left corner is cleaned inside the generator now, right?

Reply:
Yes that's it :+1: